packages/sitecore-jss-nextjs/src/utils/index.ts

```diff
@@ -0,0 +1,2 @@
+export { getPublicUrl, handleEditorFastRefresh } from './utils';
+export { tryParseEnvValue } from '@sitecore-jss/sitecore-jss/utils';
```

Comment:
Don't we need to export isEditorActive, resetEditorChromes, resolveUrl as well here (our deprecated comment is instructing them to do so).

Reply:
Also, I don't see changes in our template moving over to use the /utils submodule version of these functions. We should do that so that there is no use of the deprecated versions.

Reply:
Right, that was omitted, but should be addressed now.